next/components/forms/types/AccountTypes.ts

```diff
@@ -0,0 +1,8 @@
+import { ReactNode } from 'react'
+
+export interface SectionItemBase {
+  id: number
+  title: string
+  icon: ReactNode
+  link: string
+}
```

Comment:
this can be colocated in AccountPageLayout and exported from there if needed, no need for an extra file